config/passport.js

```diff
@@ -161,7 +172,7 @@ module.exports = (passport) => {
               username: profile.username,
               provider: 'google',
               google: profile._json,
-              avatar: profile._json.picture
+              picture: profile._json.picture
```

Comment:
This will break some other part of the code as they rely on avatar instead of the picture object. That's why the avatar was retained instead. same meaning different names.
It would be good if this is reverted.